frankenstein/prog/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/prometheus/common/log"
+	"github.com/prometheus/prometheus/frankenstein"
+)
+
+func main() {
+	var (
+		listen       string
+		consulHost   string
+		consulPrefix string
+	)
+
+	flag.StringVar(&listen, "listen", ":80", "HTTP server listen address")
+	flag.StringVar(&consulHost, "consul.hostname", "consul:8500", "Hostname & port of consul")
```

Comment:
Proper noun nit: capitalize "Consul" here and below. Avoid "&" in favor of "and", especially in user-visible docs.